drivers/source/FlashIAP.cpp

```diff
@@ -46,7 +46,7 @@ static inline bool is_aligned(uint32_t number, uint32_t alignment)
     }
 }
 
-FlashIAP::FlashIAP() : _page_buf(nullptr)
+FlashIAP::FlashIAP() : _page_buf(nullptr), _flash{nullptr}
```

Comment:
There is another PR already fixing this, see https://github.com/ARMmbed/mbed-os/pull/11494/files

Reply:
Removed from PR